### PR TITLE
BXMSDOC-1911 master Add instructions for changing settings in the Security perspective

### PR DESCRIPTION
**Pull request by emmurphy1**
The rendered doc is [here](http://file.ork.redhat.com/~emmurphy/BXMSDOC-1911/#business-central-settings-security-proc).

### CODE REVIEW

File: docs/product-user-guide/src/main/asciidoc/business-central-settings-security-con.adoc [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[id='business-central-settings-security-proc']
+= Security settings
+
+To access the Security perspective, click *Roles*, *Groups*, or *Users* on the Settings perspective. Depending on which icon you click, the Security perspective opens on the Roles, Groups, or Users tab. Use roles and groups to manage privileges and settings for users. Any changes apply to all users with the affected role or group.
```
Comment (sterobin) [inline docs/product-user-guide/src/main/asciidoc/business-central-settings-security-con.adoc]: Should also be bold in "opens in **Roles**, **Groups**, or **Users** tabs." Our internal style to date has been to bold all actionable UI elements, whether active or not. From what I understand.

File: docs/product-user-guide/src/main/asciidoc/business-central-settings-changing-permissions-proc.adoc [hunk 2]
```diff
@@ -0,0 +1,18 @@
+[id='business-central-settings-changing-permissions-proc']
+= Changing permissions
```
Comment (sterobin) [inline docs/product-user-guide/src/main/asciidoc/business-central-settings-changing-permissions-proc.adoc]: Given the limitation that you mention in the opening section, I'd change the title to "Changing permissions for groups and roles"

File: docs/product-user-guide/src/main/asciidoc/business-central-settings-creating-new-users-proc.adoc [hunk 3]
```diff
@@ -0,0 +1,15 @@
+[id='business-central-settings-creating-new-users-proc']
+= Creating new users
+You can create as many {PRODUCT} users as you require. User privileges and settings are controlled by the roles assigned to a user and the groups that a user belongs to.
+
+.Procedure
+. Click the gear icon, then click *Users*.
+. Click *New user*, enter a user name, then click *Next*.
+. If desired, click the *Groups* tab, click *Add to groups*, select the groups that you want this user to belong to, then click *Add to selected groups*.
```
Comment (sterobin) [inline docs/product-user-guide/src/main/asciidoc/business-central-settings-creating-new-users-proc.adoc]: Yeah, I see what you mean. What I do with things like this is to make two sentences, with 3 actions or less per sentence, or whatever seems logical. Still feels good in a single step, but makes it feel less packed.
So like this:
"If desired, click the **Groups** tab, click **Add to groups**, and select the groups that you want this user to belong to. Then click **Add to selected groups**."
Or something.

File: docs/product-user-guide/src/main/asciidoc/business-central-settings-setting-priorities-proc.adoc [hunk 4]
```diff
@@ -0,0 +1,10 @@
+[id='business-central-settings-setting-priorities-proc']
+= Setting priorities
+
+A user can have multiple roles and belong to multiple groups. The Priority setting determines the order of precedence of a role or group.
+
+.Procedure
+Select a priority  from the Priority menu, then click *Save*. If a user has a role or belongs to a group that has conflicting settings, the settings of the role or group with the highest priority applies.
```
Comment (sterobin) [inline docs/product-user-guide/src/main/asciidoc/business-central-settings-setting-priorities-proc.adoc]: Not a huge deal (I don't care), but IBM says to say "and then" in a series, not just "then". So "menu, and then". If you're starting a new sentence, though, like in my example a few comments up, you can start "Then". Your call.

File: docs/product-user-guide/src/main/asciidoc/proc-business-central-settings-data-sources.adoc [hunk 5]
```diff
@@ -1,4 +1,4 @@
 [id='business_central_settings_data_sources']
 = Data Sources
 
-This perspective enables you to configure connections to external databases. To connect to an external database, the required JDBC driver must be available. You can add JDBC drivers in this perspective. The data sources created in this perspective can be used to create SQL data sets. For more information, see <<data_sets_con>>.
+This perspective enables you to configure connections to external databases. To connect to an external database, the required JDBC driver must be available. You can add JDBC drivers in this perspective. The data sources created in this perspective can be used to create SQL data sets.
```
Comment (sterobin) [inline docs/product-user-guide/src/main/asciidoc/proc-business-central-settings-data-sources.adoc:4]: Thanks for snagging this!

File: docs/product-user-guide/src/main/asciidoc/business-central-settings-creating-new-groups-proc.adoc [hunk 6]
```diff
@@ -0,0 +1,10 @@
+[id='business-central-settings-creating-new-groups-proc']
+= Creating new groups
+
+Groups enable you to control privileges and settings for a collection of users. You can create as many groups as you require. 
+
+.Procedure
+. Click the gear icon, then click *Groups*.
+. Click *New group*, enter a group name, then click *Next*.
+. Select the users that you want to belong to this group, then click *Add selected users*.
```
Comment (sterobin) [inline docs/product-user-guide/src/main/asciidoc/business-central-settings-creating-new-groups-proc.adoc]: Same "and then" issue. See my note a few comments down. Again, I really don't care, personally.

File: docs/product-user-guide/src/main/asciidoc/business-central-settings-creating-new-users-proc.adoc [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+. If desired, click the *Roles* tab, click *Add roles*, select the roles that you want to give this user, then click *Add to selected roles*.
```
Comment (sterobin) [inline docs/product-user-guide/src/main/asciidoc/business-central-settings-creating-new-users-proc.adoc]: Ditto.

File: docs/product-user-guide/src/main/asciidoc/business-central-settings-creating-new-users-proc.adoc [hunk 8]
```diff
@@ -0,0 +1,15 @@
+[id='business-central-settings-creating-new-users-proc']
+= Creating new users
```
Comment (sterobin) [inline docs/product-user-guide/src/main/asciidoc/business-central-settings-creating-new-users-proc.adoc]: I'd change this and the groups section to "Creating users" and "Creating groups". To create something entails it being new, so it's redundant. This is a common issue in most documentation, and in ours. The only exception, for me, is when you want to emphasize new vs existing.

Comment (emmurphy1) [reply docs/product-user-guide/src/main/asciidoc/business-central-settings-creating-new-users-proc.adoc]: So true! Thanks for catching.

File: docs/product-user-guide/src/main/asciidoc/business-central-settings-changing-permissions-proc.adoc (hunk 2 above)
Comment (emmurphy1) [reply docs/product-user-guide/src/main/asciidoc/business-central-settings-changing-permissions-proc.adoc]: Done

**Pull request merged**